pkgs/development/tools/benthos/default.nix

```diff
@@ -0,0 +1,32 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "benthos";
+  version = "4.3.0";
+
+  src = fetchFromGitHub {
+    owner = "benthosdev";
+    repo = "benthos";
+    rev = "v${version}";
+    sha256 = "sha256-tRB9eTeyEyPkiR/sph76CMbPjJUNoDzfYuHmtFAzY7E=";
+  };
+
+  vendorSha256 = "sha256-nnaBQ7ADdAdo/+RQzXJHBBpXgTmxny0O/ij+eWsS5YM=";
+
+  proxyModule = true;
```

Comment:
This isn't a valid argument to `buildGoModule`. Did you mean `proxyVendor`? Please test building with `proxyVendor` set to true and false, and if it builds successfully when it's false, don't set it (false is the default).

Reply:
Good catch, thanks! It does work without proxyVendor.